e2e/wdio/core/tests/message-toast.e2e-spec.ts

```diff
@@ -0,0 +1,83 @@
+import { MessageToastPo } from '../pages/message-toast.po';
+import {
+    waitForElDisplayed,
+    refreshPage,
+    click,
+    isElementDisplayed,
+    pause,
+    mouseHoverElement,
+    doesItExist,
+    scrollIntoView,
+    getElementArrayLength,
+} from '../../driver/wdio';
+
+describe('Textarea component test', function () {
+    const messageToastPage = new MessageToastPo();
+    const {
+        openMessageButton,
+        hideAllButton,
+        message
+    } = messageToastPage;
+
+    beforeAll(() => {
+        messageToastPage.open();
+    }, 1);
+
+    beforeEach(() => {
+        refreshPage();
+        waitForElDisplayed(messageToastPage.title);
+    }, 2);
+
+    it('should check message from component', () => {
+        click(openMessageButton);
+        expect(isElementDisplayed(message)).toBe(true, 'message is not displayed');
+        pause(7500);
```

Comment:
I have a suggestion.. either do waitForElNoDisplayed(element, index, 7500); if you want to check it's gone in the correct time
or click the 'hide all' btn if you don't want to check the timing.

pause should only be used if absolutely needed, i.e. animation like column transition etc.